tests/test_function.py

```diff
@@ -215,6 +215,56 @@ def test_multivariable_dataset(a, b):
     assert np.isclose(func(a, b), a + b, atol=1e-6)
 
 
+@pytest.mark.parametrize(
+    "x,y,z_expected",
+    [
+        (0, 0, 1),
```

Comment:
Just noticed this test case is repeated, I will fix it soon.

Reply:
Solved by [acbc14d](https://github.com/RocketPy-Team/RocketPy/pull/515/commits/acbc14dae924d874e6a2803e2f494be92eec7f09).